### DISCUSSION TITLE
How do I get logs input from docker containers that writes to stdout?

### DISCUSSION THREAD

**Discussion by chiawendt** [fluent/fluent-bit#9774] [Q&A]
I have some containers that writes to stdout and I want to collect the logs from them, how do I do it? I think it might be possible to use tail input for `/var/lib/docker/container/**/*.log`, but I don't know if it is actually feasible.

**Comment by patrick-stephens** [comment]
Depends, are you running in K8S in which case the kubelet writes those logs to a file for you so you just tail that.

If you mean from manually run containers then you can adjust the logging driver to use an alternative and ingest it that way, e.g. https://docs.docker.com/engine/logging/drivers/fluentd/ uses the fluent forward protocol so you can ingest it with a forward input: https://docs.fluentbit.io/manual/pipeline/inputs/forward
You can also just get docker to write to log files and tail those.
